pkgs/development/python-modules/pyparser/default.nix

```diff
@@ -0,0 +1,27 @@
+{ buildPythonPackage, lib, fetchFromBitbucket
+, parse
+}:
+
+buildPythonPackage rec {
+  pname = "pyparser";
+  version = "1.0";
+
+  src = fetchFromBitbucket {
```

Comment:
Why not `fetchPypi`? `fetchPypi` is generally preferred because this makes it easier to override the `src`. However, in case e.g. tests are not uploaded to PyPI, it is better to use other sources.
The same applies for all other packages.

Reply:
for this and for papis tests are missing, for one other they are failing. Fixed others